### PR TITLE
Add scm-source to Dockerfile when building.

### PR DESCRIPTION
**Pull request by feikesteenbergen**
To be compliant with Stups we need to ensure we add an up-to-date
scm-source.json file to the root of the Docker container.

### CODE REVIEW

File: postgres-appliance/build.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+URL=$(git config --get remote.origin.url)
+STATUS=$(git status --porcelain)
+GITAUTHOR=$(git show -s --format="%aN <%aE>" "$REV")
+
+cat > scm-source.json <<__EOT__
+{
+    "url": "$URL",
+    "revision": "$REV",
+    "author": "$GITAUTHOR",
+    "status": "'$STATUS'"
```
Comment (hjacobs) [inline postgres-appliance/build.sh]: I see additional single quotes here, compare with http://docs.stups.io/en/latest/user-guide/application-development.html#scm-source-json

Comment (feikesteenbergen) [inline postgres-appliance/build.sh]: They seem unessesary, yes, however the same page also has the version with quotes:

https://github.com/zalando-stups/documentation/blame/master/user-guide/application-development.rst#L72

Will remove from this PR

Comment (hjacobs) [inline postgres-appliance/build.sh]: @feikesteenbergen thanks, will fix it in the docs..

Comment (hjacobs) [inline postgres-appliance/build.sh]: @feikesteenbergen no, the docs are right (single quotes are needed to end the outer string and to interpret the env var)

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**